webapp/store/logic.py

```diff
@@ -427,10 +425,13 @@ def get_icon(media):
     return [m["url"] for m in media if m["type"] == "icon"]
 
 
-def get_videos(media):
-    return [
-        get_video_embed_code(m["url"]) for m in media if m["type"] == "video"
-    ]
+def get_video(media):
+    video = None
+    for m in media:
+        if m["type"] == "video":
+            video = get_video_embed_code(m["url"])
```

Comment:
Could you just return here which means you don't need the `video` variable or the break and return below?